admin/data/parcelpro-shipping-settings-fields.php

```diff
@@ -125,6 +125,38 @@
         'description' => 'Customer note (billing address) invullen bij het veld "inhoud" in het verzendsysteen.',
         'desc_tip'    => true
     ),
+    'postnl_show_delivery_date' => array(
+        'type'     => 'checkbox',
+        'title'    => 'Toon Verwachte levertijd voor PostNL',
+        'label'    => 'Toon Verwachte levertijd voor PostNL',
+        'desc_tip' => true,
+        'default'  => 'no',
+        'description' => 'Toon bij de checkout de geschatte tijd waarop het pakket geleverd zal worden',
+    ),
+    'postnl_last_shipping_time' => array(
+        'type'     => 'text',
+        'title'    => 'Cutoff tijd van PostNL voor verwachte levertijd berekening.',
+        'label'    => 'Cutoff tijd van PostNL voor verwachte levertijd berekening.',
+        'default'  => '17:00 Europe/Amsterdam',
+        'desc_tip' => true,
+        'description' => 'Stel hier de tijd in waarop de bestelling geplaatst moet zijn om nog dezelfde dag verzonden te worden.',
+    ),
+    'dhl_show_delivery_date' => array(
+        'type'     => 'checkbox',
+        'title'    => 'Toon Verwachte levertijd voor DHL',
+        'label'    => 'Toon Verwachte levertijd voor DHL',
+        'desc_tip' => true,
+        'default'  => 'no',
+        'description' => 'Toon bij de checkout de geschatte tijd waarop het pakket geleverd zal worden',
+    ),
+    'dhl_last_shipping_time' => array(
+        'type'     => 'text',
+        'title'    => 'Cutoff tijd van DHL voor verwachte levertijd berekening.',
+        'label'    => 'Cutoff tijd van DHL voor verwachte levertijd berekening.',
+        'default'  => '17:00 Europe/Amsterdam',
+        'desc_tip' => true,
+        'description' => 'Stel hier de tijd in waarop de bestelling geplaatst moet zijn om nog dezelfde dag verzonden te worden.',
```

Comment:
Ik zou ook graag (ergens, hoeft niet per sé in de tooltip) een voorbeeld tijd+tijdzone zien, voor als de gebruiker for some reason het vak leeg heeft gemaakt maar wel een voorbeeld wil.

Reply:
Done, met een placeholder en in de tooltip